src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -132,6 +188,9 @@
           :activeIndex="activeIndex"
           :validate="getComponentValidate"
           :validOrgPersons="validOrgPersons"
+          :showDeliveryAddressColumn="!(isLimitedExtendRestorationFiling || isLimitedConversionRestorationFiling)"
+          :showRolesColumn="isBenBcCccUlcCorrectionFiling"
+          :showEmailColumn="isLimitedExtendRestorationFiling || isLimitedConversionRestorationFiling"
```

Comment:
This is a good example of a sub-component (the list) not needing to know what type of entity/filing this is. Ultimately we might move these rules to a resource file, but not today. Looks good.